File: PrimeNumberMath.java
import java.util.Scanner;

public class PrimeNumberMath {

    public static boolean isPrime(int num) {
        if (num < 2) {
            return false;
        }
        for (int i = 2; i <= Math.sqrt(num); i++) {
            if (num % i == 0) {
                return false;
            }
        }
        return true;
    }

    public static int nthPrime(int n) {
        int count = 0;
        int i = 2;
        while (count < n) {
            if (isPrime(i)) {
                count++;
            }
            i++;
        }
        return i - 1;
    }

    public static int addPrimes(int pos1, int pos2) {
        int prime1 = nthPrime(pos1);
        int prime2 = nthPrime(pos2);
        return prime1 + prime2;
    }

    public static int subtractPrimes(int pos1, int pos2) {
        int prime1 = nthPrime(pos1);
        int prime2 = nthPrime(pos2);
        return prime1 - prime2;
    }

    public static int multiplyPrimes(int pos1, int pos2) {
        int prime1 = nthPrime(pos1);
        int prime2 = nthPrime(pos2);
        return prime1 * prime2;
    }

    public static int dividePrimes(int pos1, int pos2) {
        int prime1 = nthPrime(pos1);
        int prime2 = nthPrime(pos2);
        return prime1 / prime2;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the position of the first prime number: ");
        int pos1 = scanner.nextInt();
        System.out.print("Enter the position of the second prime number: ");
        int pos2 = scanner.nextInt();

        int sum = addPrimes(pos1, pos2);
        int difference = subtractPrimes(pos1, pos2);
        int product = multiplyPrimes(pos1, pos2);
        int quotient = dividePrimes(pos1, pos2);

        System.out.println("The sum is: " + sum);
        System.out.println("The difference is: " + difference);
        System.out.println("The product is: " + product);
        System.out.println("The quotient is: " + quotient);
    }
}

File: PrimeNumberMath.py
import math

def is_prime(n: int) -> bool:
    """Return True if n is prime, else False."""
    if n <= 1:
        return False
    for i in range(2, int(math.sqrt(n))+1):
        if n % i == 0:
            return False
    return True

def nth_prime(n: int) -> int:
    """Return the nth prime number."""
    count = 0
    i = 2
    while True:
        if is_prime(i):
            count += 1
            if count == n:
                return i
        i += 1

def add_primes(a: int, b: int) -> int:
    """Return the sum of the a-th and b-th prime numbers."""
    return nth_prime(a) + nth_prime(b)

def subtract_primes(a: int, b: int) -> int:
    """Return the difference of the a-th and b-th prime numbers."""
    return nth_prime(a) - nth_prime(b)

def multiply_primes(a: int, b: int) -> int:
    """Return the product of the a-th and b-th prime numbers."""
    return nth_prime(a) * nth_prime(b)

def divide_primes(a: int, b: int) -> float:
    """Return the quotient of the a-th and b-th prime numbers."""
    return nth_prime(a) / nth_prime(b)

if __name__ == '__main__':
    a = int(input("Enter the position of the first prime number: "))
    b = int(input("Enter the position of the second prime number: "))
    
    print(f"{a}-th prime number: {nth_prime(a)}")
    print(f"{b}-th prime number: {nth_prime(b)}")
    print(f"Sum: {add_primes(a, b)}")
    print(f"Difference: {subtract_primes(a, b)}")
    print(f"Product: {multiply_primes(a, b)}")
    print(f"Quotient: {divide_primes(a, b)}")
